backend/dataall/core/permissions/db/tenant/tenant_policy_repositories.py

```diff
@@ -0,0 +1,113 @@
+import logging
+
+from sqlalchemy.sql import and_
+
+from dataall.core.permissions.api.enums import PermissionType
+from dataall.base.db import exceptions, paginate
+from dataall.core.permissions.constants import permissions
+from dataall.core.permissions.db.permission import permission_models
+from dataall.core.permissions.db.tenant import tenant_models as models
+from dataall.core.permissions.db.permission.permission_repositories import Permission
+
+logger = logging.getLogger(__name__)
+
+
+class TenantPolicy:
+    TENANT_NAME = 'dataall'
```

Comment:
Define `DAAdministrators` as constant instead of hardcoding it in line 102

Reply:
Good catch. Also, I used in in the TenantPolicyValidationService